modules/host/negotiaterenewcontract.go

```diff
@@ -241,7 +241,7 @@ func (h *Host) managedVerifyRenewedContract(so *storageObligation, txnSet []type
 	}
 	// The WindowEnd for the new file contract must be further in the future
 	// than the WindowEnd for the existing file contract.
-	if fc.WindowEnd <= so.proofDeadline() {
+	if fc.WindowEnd < so.proofDeadline() {
```

Comment:
Does it mess things up too badly if you just increase the height by '1' every time?